x-pack/plugins/alerting/server/rule_type_registry.ts

```diff
@@ -315,7 +314,7 @@ export class RuleTypeRegistry {
           spaceId: schema.string(),
           consumer: schema.maybe(schema.string()),
         }),
-        indirectParamsSchema: rawRuleSchemaV1,
+        indirectParamsSchema: ruleType.validate.params,
```

Comment:
Why are we no longer validating the rule schema and just the rule parameters? Seems like changes to the rule schema could also potentially be backwards incompatible

Reply:
And the rule schema is just used for modelVersions which validates the rule just before creating the SO in ES.
This is to force developers to update the schema and bump the version.